### DISCUSSION TITLE
Settings button default missing in menu in version 1.3.0.0

### DISCUSSION THREAD

**Discussion by ShermanBob** [Hofknecht/SystemTrayMenu#413] [Q&A] [answered]
**Description, Steps to reproduce, Expected behavior, Screenshots**
I don't know how it happened, but this morning the icon for the app changed to the ">>" characters on a black background (vs. the multi-color old icon) and I no longer see the settings button upon invoking the app's popup folder navigation interface.  I suspect an auto-upgrade happened, but I do not know how that would happen.
 - OS: [e.g. Microsoft Windows 10 Pro]:
 - Version [e.g. 0.9.1.233]

**Comment by Hofknecht** [comment, marked as answer]
@ShermanBob 
Thank you for the feedback!

Do you use portable version or Microsoft Store version?
in version 1.3.0.0:
-the icon is  ">>"
-the settings key function is default off but can be enabled via settings
![2022-07-13 23_09_06-Window](https://user-images.githubusercontent.com/52528841/178836707-d323554c-6f59-48a5-b8ad-42ca71742838.png)
-there is no auto upgrade normally (maybe only via Microsoft Store) only a version check can be enabled

Which version is now installed? You can check this via about menu.

Maybe can you give us more hints about the issue and add a screenshot and tell us more about your expectiations?

Thank you in advanced!


Find the settings menu via systemtray icon, right click, context menu
![2022-07-14 09_56_27-](https://user-images.githubusercontent.com/52528841/178931859-9f39f7f3-188d-4ff2-914e-1422ffbd59f6.png)

**Comment by Hofknecht** [comment]
@ShermanBob 
i will convert into a discussion -> question answer
